### PR TITLE
sample_misc: Addin WAYLAND DMABUF support to render NV12 on Wayland.

### PR DESCRIPTION
**Pull request by UngTengEn**
According to Vivek, the MESA iris driver expected 2 planes for NV12 and wl_drm will be deprecated in the future.  Change to use Wayland's linux-dmabuf protocol in this case.

### CODE REVIEW

**Comment by UngTengEn** [comment]
Ung, Teng En would like to recall the message, "[Intel-Media-SDK/MediaSDK] sample_misc: Addin WAYLAND DMABUF support to render NV12 on Wayland. (#2803)".

**Review by UngTengEn — COMMENTED** [review]
Reviewed

File: samples/sample_misc/wayland/include/linux-dmabuf-unstable-v1.h [hunk 1]
```diff
@@ -0,0 +1,605 @@
+/* Generated by wayland-scanner 1.18.0 */
```
[the remark this answers is not in the document]
Comment (UngTengEn) [reply samples/sample_misc/wayland/include/linux-dmabuf-unstable-v1.h:1]: Here is the steps 
1. wayland-scanner "client-header" /usr/share/wayland-protocols/unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml linux-dmabuf-unstable-v1.h
2. wayland-scanner "private-code" /usr/share/wayland-protocols/unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml linux-dmabuf-unstable-v1.c

**Comment by UngTengEn** [comment]
Hi @dmitryermilov,

     Is thrre any update on the merge request?


Thanks

**Comment by dmitryermilov** [comment]
Hi @VUNNYSODHI , sorry for delayed reply. 
I'd propose to promote  https://github.com/UngTengEn/MediaSDK/commits/dev_wayland_render_nv12 version. Can you please update the PR?

**Comment by UngTengEn** [comment]
Hi,
       I have file a new PR for https://github.com/UngTengEn/MediaSDK/commits/dev_wayland_render_nv12 [here](https://github.com/Intel-Media-SDK/MediaSDK/pull/2826).
      Then, I will close this.

Thanks

**Pull request closed without merging**